File: OpenApi2.0/push-openapi-net-demo/BindTag.cs
using Aliyun.Acs.Core;
using Aliyun.Acs.Core.Exceptions;
using Aliyun.Acs.Core.Profile;
using Aliyun.Acs.Push.Model.V20160801;
using System;

namespace AlibabaCloud
{
    class BindTag
    {
        static void Main()
        {
            IClientProfile clientProfile = DefaultProfile.GetProfile("cn-hangzhou", "<your access key id>", "<your access key secret>");
            DefaultAcsClient client = new DefaultAcsClient(clientProfile);
            BindTagRequest request = new BindTagRequest();
            request.AppKey = <Your AppKay>;
            request.KeyType = "DEVICE";//设备：DEVICE 账号：ACCOUNT 别名：ALIAS
            request.ClientKey = <Your DeviceIds>;//一次最多可以绑定1000个Key到10个tag上
            request.TagName = "tag1,tag2";//一次最多可以绑定1000个Key到10个tag上

            try
            {
                BindTagResponse response = client.GetAcsResponse(request);
                Console.WriteLine("RequestId:" + response.RequestId);
                Console.ReadLine();
            }
            catch (ServerException e)
            {
                Console.WriteLine(e.ErrorCode);
                Console.WriteLine(e.ErrorMessage);
                Console.ReadLine();
            }
            catch (ClientException e)
            {
                Console.WriteLine(e.ErrorCode);
                Console.WriteLine(e.ErrorMessage);
                Console.ReadLine();
            }
        }

    }
}

File: OpenApi2.0/push-openapi-php-demo/BindTag.php
<?php

include_once '../aliyun-php-sdk-core/Config.php';
include 'Push/Request/V20160801/BindTagRequest.php';
use \Push\Request\V20160801 as Push;

$accessKeyId = "your accessKeyId";
$accessKeySecret = "your accessKeySecret";
$appKey = "your appKey";
$deviceId = "your deviceId";
$account = "your account";

$iClientProfile = DefaultProfile::getProfile("cn-hangzhou", $accessKeyId, $accessKeySecret);
$client = new DefaultAcsClient($iClientProfile);
$request = new Push\BindTagRequest();

$request->setAppKey($appKey);
$request->setKeyType("DEVICE");//DEVICE：是设备，ACCOUNT：是账号，ALIAS：是别名
$request->setClientKey($deviceId);
$request->setTagName("tagTest");

$response = $client->getAcsResponse($request);
print_r("\r\n");
print_r($response);

?>
